code/modules/assembly/flash.dm

```diff
@@ -71,13 +80,13 @@
 /obj/item/assembly/flash/proc/burn_out() //Made so you can override it if you want to have an invincible flash from R&D or something.
 	if(!burnt_out)
 		burnt_out = TRUE
+		if(ismob(loc))
+			var/mob/M = loc
+			M.visible_message("<span class='danger'>[src] burns out!</span>","<span class='userdanger'>[src] burns out!</span>")
+		else
+			var/turf/T = get_turf(src)
+			T.visible_message("<span class='danger'>[src] burns out!</span>")
```

Comment:
I know you didn't write this, but `visible_message` is something on `/atom`, this should just be `loc?.visible_message`